lib/assesments/assessment_2/fi-as2-hira/Screens/Home/widgets/fi_as2_hira_stack_avatar.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+
+class H1Stackcircle extends StatelessWidget {
```

Comment:
Create one widget `H1Avatar`
Then pass in some booleans to adjust this approach.

You don't need to have separate widget for each avatar situation.